defs/scenarios/storage/ceph/ceph-mon/PG_overdose.yaml

```diff
@@ -0,0 +1,31 @@
+checks:
+  pg_pending_creation:
+    input:
+      command: ceph_health_detail
+    expr: '.* PENDING_CREATING_PGS: .+ PGs pending on creation'
+    #expr: 'creation'
+  health_warning:
+    property:
+      path: hotsos.core.plugins.storage.ceph.CephCluster.health_status
+      ops: [[ne, HEALTH_OK]]
+  # this one will run on OSD sos
+  ceph_osd_withhold_creation:
+    input:
+      path: var/log/ceph/ceph*.log
+    search:
+      expr: '(.+) maybe_wait_for_max_pg withhold creation of pg (.+): .*'
+
+conclusions:
+  pending_creating_pgs:
+    decision:
+      - health_warning
+      - or:
+          - pg_pending_creation
+          - ceph_osd_withhold_creation
+    raises:
+      type: CephWarning
+      message: >-
+        Ceph is reporting that you seem to have hit the hard limit
+        for the number of PGs per OSD. The mon_max_pg_per_osd may
```

Comment:
Can also increase `osd_max_pg_per_osd_hard_ratio` as a workaround.

Reply:
Also if the PGs to be created is too high (say for example 5000), then we probably want to investigate that first before increasing the limits to workaround.

Reply:
I've updated the messages and tests with more reasonable messages but not added the message to check more in detail if the value is double the limit, because its weird to print the values but then ask the user to do the "if" manually. I've just changed the messages to what I felt is a reasonable message to flag this situation.